mcodingbot/plugins/stats.py

```diff
@@ -20,6 +20,23 @@
 plugin = Plugin()
 
 
+@plugin.include
+@crescent.command(
+    name="stats", description="Exact values for mCoding statistics"
+)
+async def stats(ctx: Context) -> None:
+    embed = hikari.Embed(
+        title="mCoding stats",
+        color=CONFIG.theme,
+        description=(
+            f"Server members: `{ctx.server.member_count:,}`\n"
```

Comment:
None of this exists. Please run your code before making a PR.

Reply:
@FuexFollets you can use `_last_known_stats.subs` and `_last_known_stats.views` for sub and view counts.

For the member count, lines 71-79 of this file are how it should be done. I would make that a function in this file to avoid code duplication.